torchao/prototype/moe_training/benchmarks/benchmark_moe_layer.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
nit: move to `benchmarks/prototype/moe_training`, to match existing location of benchmarks

Reply:
done, moved all moe training benchmark scripts in this PR stack there